VERSIONING.md

```diff
@@ -1,13 +1,41 @@
-is designed so that the following goal can be achieved:
+The OpenTelemetry Collector SIG produces several artifacts for [a variety of audiences](CONTRIBUTING.md#target-audiences). This document describes the versioning and support policy for these artifacts. These policies are designed so that the following goal can be achieved:
 
-**Users are provided a codebase of value that is stable and secure.**
+**Users are provided software artifacts of value that are stable and secure.**
 
-## Public API expectations
+The policies are divided depending on the artifact's target audience. While an artifact is supported, [critical bugs](docs/release.md#bugfix-release-criteria) and security vulnerabilities MUST be addressed. The main criteria for the length of support for an artifact is  how easy it is for an artifact's target audience to adapt to disruptive changes.
 
-The following public API expectations apply to all modules in opentelemetry-collector and opentelemetry-collector-contrib.
+## End-user focused software artifacts
+
+End-user focused software artifacts are those that are intended to be used by [end-users](CONTRIBUTING.md#end-users) of the OpenTelemetry Collector. These artifacts include 
+- Binary distributions of the OpenTelemetry Collector.
+- Collector components.
+
+These artifacts are versioned according to the [semantic versioning v2.0.0](https://semver.org/) specification.
+
+### General considerations
+
+Binary distributions produced by the Collector SIG contain components and features with varying [levels of stability](README.md#stability-levels). We abide by the following principles to relate the Collector's version to the stability of its components and features:
+
+* The Collector's core framework MUST be stable in order for a Collector distribution to be v1.0.0 or higher.
+* Users can easily understand when they are opting in to use a component or feature that is not stable.
+    * The Collector MUST provide a configuration or CLI flag to be able to opt out of using unstable components or features.
+    * The Collector's telemetry (e.g. Collector logs) MUST provide the ability to identify usage of unstable components or features.
+
+### Long term support
+
+The OpenTelemetry Collector SIG provides long term support for binary distributions of the OpenTelemetry Collector and its components. The following policies apply to long term support for any major version starting on v1:
+
+* A binary distribution of the OpenTelemetry Collector MUST be supported for a minimum of **one year** after the release of the next major version of said distribution.
+* Components MUST be supported for a minimum of **6 months** after the release of the next major version of said component or after the component has been marked as deprecated.
+* Components MAY be removed from a binary distribution of the OpenTelemetry Collector after the component has been marked as deprecated for a minimum of **6 months**.
```

Comment:
Just to be clear, this is elaborating on the previous point and describes a stable version of a component being removed from a distribution with a beta version?

Reply:
I merged it into a single point. The intent is to say that we can remove components from v1 distros, provided we have deprecated the components before